addons/sale/models/sale_order.py

```diff
@@ -227,7 +223,9 @@ def _search_invoice_ids(self, operator, value):
         ('no', 'Nothing to Invoice')
         ], string='Invoice Status', compute='_get_invoice_status', store=True)
 
-    note = fields.Html('Terms and conditions', default=_default_note)
+    note = fields.Html(
+        'Terms and conditions', default=_default_note,
+        compute='_compute_note', store=True, readonly=False)
```

Comment:
pre_compute=True & remove the default?

Currently, the default note won't be computed with the partner language, even if the partner is given as create values.